models/category.js

```diff
@@ -17,7 +18,7 @@ const categorySchema = new Schema(
     },
     totalOffers: {
       type: Number,
-      ref: 'Offer',
+      ref: OFFER,
```

Comment:
what about exports? use there consts too? the last line in every model file?

Reply:
okey, added to models name constants too